androidApp/src/main/res/values/themes.xml

```diff
@@ -36,4 +36,13 @@
         <item name="windowSplashScreenAnimatedIcon">@mipmap/ic_launcher</item>
         <item name="postSplashScreenTheme">@style/Theme.ProjectNewm.NoActionBar</item>
     </style>
+
+    <style name="Theme.ProjectNewm.SplashScreen.NoActionBar">
+        <item name="windowActionBar">false</item>
```

Comment:
Do we want to just add these to `Theme.ProjectNewm.SplashScreen`? I wouldn't think we will ever need both

Reply:
BTW this is what switches the theme after splash: 
```<item name="postSplashScreenTheme">@style/Theme.ProjectNewm.NoActionBar</item>```

Reply:
I can make that part of the next set of changes 👍🏼